### PR TITLE
feat(server):swagger support auth for standardAuth mode 

### PR DESCRIPTION
**Pull request by SunnyBoy-WYH**
…x arthas odd test

## Purpose of the PR

swagger support auth for standardAuth mode and try to fix arthas odd test
- subtask if #2088 

## Main Changes
1. add swagger auth annotation.
2. set openapi.json to whitelist.
3. try to simplify arthas test,try to fix the odd test failed case.

## Verifying these changes

no auth model,we can try it out in swagger:
24fbd7022f9f6a3ff361442ab6477bb
 auth model,we cant try it out in swagger:
1d4bec0ddee89e86e5eef603b021dd1
after set auth info,success:
9c4b6f83e3c7e7ef4b02cad243f50b3

559b6f513d241a487f3d903c682d283

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [ ]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [ ]  `Doc - No Need`

### CODE REVIEW

File: hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/filter/AuthenticationFilter.java [hunk 1]
```diff
@@ -72,7 +72,8 @@ public class AuthenticationFilter implements ContainerRequestFilter {
 
     private static final List<String> WHITE_API_LIST = ImmutableList.of(
             "auth/login",
-            "versions"
+            "versions",
+            "openapi.json"
```
Comment (imbajin) [inline hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/filter/AuthenticationFilter.java:76]: we add swagger file to white_list by default to allow normal users to check the api & not locked by auth

File: hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/server/ApplicationConfig.java [hunk 2]
```diff
@@ -44,16 +44,25 @@
...
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.security.SecurityScheme;
 import jakarta.ws.rs.ApplicationPath;
 
+@SecurityScheme(
+    name = "basic",
+    type = SecuritySchemeType.HTTP,
+    scheme = "basic"
+)
```
Comment (imbajin) [inline hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/server/ApplicationConfig.java:56]: TODO: if users need `token` mode in future, we could add the option like this (now we just enable basic auth)

68c58c71fa3454a84c07963a8aaaa8a0

**Comment by imbajin** [comment]
One more thing, we need update apiversion for metrics & arthas API here:
image

**Comment by SunnyBoy-WYH** [comment]
> One more thing, we need update apiversion for metrics & arthas API here: (use 0.70 for them maybe?) image
> 
> Also search `0.69` in all files and also change them to `0.70`

but seems we dont have related issue to direct it, can we dont use this format? 
like `[0.69] Issue-1748: Support Cypher query RESTful API`

**Comment by imbajin** [comment]
> > One more thing, we need update apiversion for metrics & arthas API here: (use 0.70 for them maybe?) image
> > 
> > Also search `0.69` in all files and also change them to `0.70`
> 
> but seems we dont have related issue to direct it, can we dont use this format? 
> like `[0.69] Issue-1748: Support Cypher query RESTful API`

issue＝ pr （number）here，and normally we need a issue before submit a pr 

but here we could just use pr number as the issue number

File: hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/server/ApplicationConfig.java (hunk 2 above)
Comment (javeme) [reply hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/server/ApplicationConfig.java:56]: can we keep the basic and token at the same time

**Comment by javeme** [comment]
please also supplement apiversion for https://github.com/apache/incubator-hugegraph/pull/2242

**Comment by SunnyBoy-WYH** [comment]
> [0.69] Issue-1748: Support Cypher query RESTful API

seems [0.70] `PR-999:Support Cypher query RESTful API` OR `ISSUE-999:Support Cypher query RESTful API`,which 999 means pr number. seems the first is better

**Comment by imbajin** [comment]
> > [0.69] Issue-1748: Support Cypher query RESTful API
> 
> seems [0.70] `PR-999:Support Cypher query RESTful API` OR `ISSUE-999:Support Cypher query RESTful API`,which 999 means pr number. seems the first is better

both fine to me（#number is also acceptable but not sure others opinion），u could try modify it

File: hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/server/ApplicationConfig.java (hunk 2 above)
Comment (SunnyBoy-WYH) [reply hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/server/ApplicationConfig.java:56]: Done
image

**Review by imbajin — APPROVED** [review]

**Review by chengxin1374 — APPROVED** [review]

**Review by simon824 — APPROVED** [review]

**Pull request merged**